commands/FBDelay.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python
+import string
+import lldb
+import fblldbbase as fb
+import fblldbobjcruntimehelpers as runtimeHelpers
+from threading import Timer
+
+
+def lldbcommands():
+  return [
+    FBDelay()
+  ]
+
+class FBDelay(fb.FBCommand):
+  def name(self):
+    return 'zzz'
+
+  def description(self):
+    return 'Executes specified lldb command after delay.'
+
+  def options(self):
+    return []
+
+  def args(self):
+    return [
+      fb.FBCommandArgument(arg='delay in seconds', type='float', help='time to wait before executing specified command'),
+      fb.FBCommandArgument(arg='lldb command', type='string', help='another lldb command to execute after specified delay', default='process interrupt')
+    ]
+
+  def run(self, arguments, options):
+    lldb.debugger.SetAsync(True)
```

Comment:
I spelunked last night, and the main thing is that the async setting decides whether `Resume()` or `ResumeSynchronous()` is called. I didn't write any code though which would really answer the question. I don't know if those other cases really confirm it either way.

Since it must work as is, we can leave this for another day 👍